### PR TITLE
added navbar

### PR DESCRIPTION
**Pull request by shivams10**

### CODE REVIEW

File: sass/main.css [hunk 1]
```diff
@@ -1 +1,118 @@
...
+     -moz-user-select: none;
+      -ms-user-select: none;
+          user-select: none;
+}
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+    position: absolute;
+    background: linear-gradient(45deg, #F6B443, #cd8201);
```
Comment (siddhi-rajput-22) [inline sass/main.css:90]: use $gradient variable here

File: sass/main.css [hunk 2]
```diff
@@ -1 +1,118 @@
...
+body li {
+  list-style: none;
+}
+
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+  background: linear-gradient(45deg, #F6B443, #cd8201);
```
Comment (siddhi-rajput-22) [inline sass/main.css:23]: use $gradient variable here

File: sass/main.css [hunk 3]
```diff
@@ -1 +1,118 @@
...
\ No newline at end of file
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  background-color: #111111;
```
Comment (siddhi-rajput-22) [inline sass/main.css:9]: use $primary-color variable here

Comment (shivams10) [reply sass/main.css:9]: actually these are .css file and i am working on .scss file,
.scss files are managed automatically by vs code extension.

**Pull request merged**